atlasdb-config/src/test/java/com/palantir/atlasdb/http/AtlasDbHttpClientsTest.java

```diff
@@ -47,15 +47,14 @@
     private static final Optional<SSLSocketFactory> NO_SSL = Optional.absent();
     private static final String TEST_ENDPOINT = "/number";
     private static final MappingBuilder ENDPOINT_MAPPING = get(urlEqualTo(TEST_ENDPOINT));
-    private static final int AVAILABLE_PORT = 8080;
     private static final int UNAVAILABLE_PORT = 8081;
```

Comment:
This might actually cause a problem on the off chance someone decides to pick 8081. I think this should be a dynamic port as well.